Add The Ship: Murder Party egg (Windows server under Wine) (#295)
pelican-eggs/games-steamcmd: the_ship/README.md

# The Ship: Murder Party

## From their [Steam Store page](https://store.steampowered.com/app/2400/The_Ship_Murder_Party/)

The Ship is a multiplayer murder-mystery game built on the Source engine. Each player is given 
a Quarry to kill – and must evade their own Hunter in the process, all set on board a series 
of 1920s art deco cruise ships.

### Server Ports

The Ship requires a single game port. It can be changed to any other.

| Port | default |
|------|---------|
| Game | 27015   |

> ## Steam Download [SteamStore](https://store.steampowered.com/app/2400/The_Ship_Murder_Party/)

Reviewer: I don't think this is needed:
```suggestion

```

## Notes

- **Runs the Windows server under Wine.** The Ship's native Linux dedicated server is broken
  on modern systems: it crashes during map load:
  - `The futex facility returned an unexpected error code` abort on glibc >= 2.34)
  - or some unexplained memory corruption if ran with a lower glibc version

> - You cannot issue commands over the panel, but you can still view logs, since scrds over wine

Reviewer: ```suggestion
- You cannot issue commands via the console due to the server being wrapped under Wine, but you can still view logs.
```

>   has compatibility issues.

Reviewer: ```suggestion

```

> - A licensed Steam account is required.

Reviewer: Is it more accurate to say this?
```suggestion
- A Steam account that owns the game is required to download the server.
```

Author: IIRC steamcmd uses the word licensed. "that owns the game" is probably more user-facing friendly indeed.
